.github/workflows/integration-tests.yml

```diff
@@ -145,3 +145,66 @@ jobs:
           /home/runner/.cargo/bin/sccache --show-stats
 
           /home/runner/.cargo/bin/sccache --show-stats | grep -e "Cache hits\s*[1-9]"
+
+  azblob_azurite:
+    runs-on: ubuntu-latest
+    needs: build
+
+    # Setup azurite server
+    services:
+      azurite:
+        image: mcr.microsoft.com/azure-storage/azurite
+        ports:
+          - 10000:10000
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+
+      - name: Setup test bucket
+        run: |
+          az storage container create \
+              --name test \
+              --connection-string "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
+
+      - name: Install rust
+        uses: ./.github/actions/rust-toolchain
+        with:
+          toolchain: "stable"
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: integration-tests
+          path: /home/runner/.cargo/bin/
+      - name: Chmod for binary
+        run: chmod +x /home/runner/.cargo/bin/sccache
+
+      - name: Test
+        run: cargo clean && cargo build
+        env:
+          RUST_BACKTRACE: full
+          RUST_LOG: debug
+          RUSTC_WRAPPER: /home/runner/.cargo/bin/sccache
+          SCCACHE_AZURE_BLOB_CONTAINER: "test"
+          SCCACHE_AZURE_CONNECTION_STRING: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
+
+      - name: Output
+        run: |
+          /home/runner/.cargo/bin/sccache --show-stats
+
+          /home/runner/.cargo/bin/sccache --show-stats | grep azblob
+
+      - name: Test Twice for Cache Read
+        run: cargo clean && cargo build
+        env:
+          RUST_BACKTRACE: full
+          RUST_LOG: debug
+          RUSTC_WRAPPER: /home/runner/.cargo/bin/sccache
+          SCCACHE_AZURE_BLOB_CONTAINER: "test"
+          SCCACHE_AZURE_CONNECTION_STRING: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
```

Comment:
Can the login and password be moved into a single place? Thanks

Reply:
What's the meaning of `moving into a single place`?

Reply:
DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1; 

is duplicated 3 times in this file ;)

Reply:
I got it~